### PR TITLE
Apply minor improvements to README

### PR DESCRIPTION
**Pull request by PawelLipski**
Some minor changes before 0.4.4 is released

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -180,8 +183,8 @@ class ExampleSerializer(actorSystem: ExtendedActorSystem)
   extends CirceAkkaSerializer[MySerializable](actorSystem) {
     // ...
     override lazy val codecs = Seq(Register[CommandOne]) // WHOOPS someone forgot to register CommandTwo...
-    // ... but Codec Registration Checker will throw an error here!
-    // ...
+    // ... but Codec Registration Checker will throw a compilation error here:
+    // `No codec for `CommandOne` is registered in class annotated with @org.virtuslab.ash.annotation.Serializer`
 }
```
Comment (PawelLipski) [inline README.md:188]: @MarconZet do you think it's worthwhile mentioning the pdf somehow from the README?

Comment (PawelLipski) [reply README.md:188]: (docs/presentation.pdf, added by this PR)

File: README.md [hunk 2]
```diff
@@ -120,10 +120,13 @@ ashDumpPersistenceSchemaOutputDirectoryPath := "~" // Changes directory
 
 ### 3. Serializer
 
-[Circe-based](https://circe.github.io/circe/) Akka serializer. It uses Circe codecs, derived using Magnolia, that are
```
Comment (PawelLipski) [inline README.md:123]: We don't even have Magnolia on the classpath AFAICS 🤔

File: project/plugins.sbt [hunk 3]
```diff
@@ -4,3 +4,4 @@ addSbtPlugin("org.scalameta" % "sbt-scalafmt" % "2.4.3")
 addSbtPlugin("ch.epfl.scala" % "sbt-scalafix" % "0.9.31")
 addSbtPlugin("com.geirsson" % "sbt-ci-release" % "1.5.7")
 addSbtPlugin("com.thoughtworks.sbt-api-mappings" % "sbt-api-mappings" % "3.0.0")
+addDependencyTreePlugin
```
Comment (MarconZet) [inline project/plugins.sbt:7]: Some leftover?

File: README.md (hunk 2 above)
Comment (PawelLipski) [reply README.md:123]: Circe comes from, ekhm, different side of the _Great Divide_ in the Scala community, they're unlikely to have picked Magnolia as their default choice

Comment (PawelLipski) [reply README.md:123]: (although there's some half-maintained https://github.com/vpavkin/circe-magnolia)

File: README.md (hunk 1 above)
Comment (MarconZet) [reply README.md:188]: I think that after sphere-it we will add a section with the talk, and put the reference there.

File: README.md (hunk 2 above)
Comment (MarconZet) [reply README.md:123]: Nice catch, did not notice it.

**Review by MarconZet — CHANGES_REQUESTED** [review]

**Review by MarconZet — APPROVED** [review]

**Pull request merged**